### PR TITLE
Add `keybase team add-member` command

### PR DESCRIPTION
**Pull request by patrickxb**

### CODE REVIEW

File: go/client/cmd_team_add_member.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	}
+
+	if err = cli.TeamAddMember(context.Background(), arg); err != nil {
+		return err
+	}
+
+	dui := c.G().UI.GetDumbOutputUI()
+
+	// send a chat message to the user telling them they are now a member of the team.
+	h := newChatServiceHandler(c.G())
```
Comment (maxtaco) [inline go/client/cmd_team_add_member.go]: Should this be in the service? What if we want to do this from electron etc?

File: go/client/cmd_team_add_member.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+	libkb.Contextified
+	team     string
+	username string
+	role     keybase1.TeamRole
+}
+
+func newCmdTeamAddMember(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "add-member",
+		ArgumentHelp: "<team name> --user=<username> --role=<owner|admin|writer|reader>",
```
Comment (patrickxb) [inline go/client/cmd_team_add_member.go]: I was just following the MVP sandbox doc.  I think this is fine for now.  

There is a service endpoint for the full ChangeMembership where multiple can be added/moved/removed in one call.

File: go/client/cmd_team_add_member.go (hunk 1 above)
Comment (patrickxb) [inline go/client/cmd_team_add_member.go]: I thought of this as an MVP-only feature.  

Chris said that in the future, there would be an email or some GUI notification that announces/manages team invitations.

File: go/client/cmd_team_add_member.go (hunk 2 above)
Comment (maxtaco) [inline go/client/cmd_team_add_member.go]: ok

File: go/client/cmd_team_add_member.go (hunk 1 above)
Comment (maxtaco) [inline go/client/cmd_team_add_member.go]: but if we just moved this function into the service, we'd anticipate that change.  in general i don't like having any interesting business logic in client.

Comment (maxtaco) [inline go/client/cmd_team_add_member.go]: as i remember, we got in trouble doing this for chat, when too much business logic was in client

**Comment by patrickxb** [comment]
PTAL

**Pull request merged**